socs/agents/scpi_psu/agent.py

```diff
@@ -68,6 +94,7 @@ def monitor_output(self, session, params=None):
                 Defaults to False.
 
         """
+        session.set_status('running')
```

Comment:
This no longer needs to be set as of [ocs v0.11.0](https://github.com/simonsobs/ocs/releases/tag/v0.11.0), remove.